tests/comparisons/Template/testBakeTemplate.ctp

```diff
@@ -8,33 +8,37 @@
         <li><?= $this->Html->link(__('List Articles'), ['controller' => 'Articles', 'action' => 'index']) ?> </li>
         <li><?= $this->Html->link(__('New Article'), ['controller' => 'Articles', 'action' => 'add']) ?> </li>
     </ul>
-</div>
-<div class="templateTaskComments view large-10 medium-9 columns">
-    <h2><?= h($templateTaskComment->id) ?></h2>
+</nav>
+<div class="templateTaskComments view large-9 medium-8 columns" id="content">
```

Comment:
I see that class `templateTaskComments` was already used earlier but it should be switched to dashed version for consistency.